packages/@okta/vuepress-site/docs/guides/add-an-external-idp/add-okta-redirect-uri-to-idp/azure/redirecttoidp.md

```diff
@@ -0,0 +1,7 @@
+1. Back in Azure, locate the client app that you created in the <GuideLink link="../create-an-app-at-idp">first section</GuideLink>.
+
+2. On the app **Overview** page, click **Add a Redirect URI**.
+
+3. Leave **Web** as the **Type**, and then paste the redirect URI that you copied in the <GuideLink link="../configure-idp-in-okta">last step</GuideLink>.
```

Comment:
Should "redirect URI" be capitalized as that's how it's cased in the UI reference?

Reply:
No, it's capitalized because it's a UI reference - and not capitalized in the next sentence because it's just a thing, but not a SPECIAL thing. Just a thing. :)